app/src/test/java/org/simple/clinic/summary/teleconsultation/contactdoctor/ContactDoctorUpdateTest.kt

```diff
@@ -32,4 +34,25 @@ class ContactDoctorUpdateTest {
             hasNoEffects()
         ))
   }
+
+  @Test
+  fun `when teleconsult request is created, then load patient teleconsult info`() {
+    val teleconsultRecordId = UUID.fromString("64ee25e0-8d32-42f5-b3e9-1a34f80e8e56")
+    val doctorPhoneNumber = "+911111111111"
```

Comment:
Here? we've added `doctorPhoneNumber` as global properties now but haven't removed them from tests?